peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -116,21 +116,21 @@ <h5>{% translate 'Related documents' %}</h5>
 
                           <ul class="list-unstyled">
                             {% for rel in document.relationships_as_subject %}
-                              {% if rel.object_document %}
+                              {% if rel.object_work %}
```

Comment:
Nice, we didn't previously have work details or FRBR URIs, so this should be better.